lib/gulp-tasks/patterns-import.js

```diff
@@ -0,0 +1,72 @@
+/**
+ *  @fileOverview
+ * Uses Gulpjs to import html patterns
+ *
+ *  @author       Scott Nath
+ *
+ *  @requires     NPM:gulp
+ *  @requires     NPM:lodash.merge
+ *  @requires     NPM:pattern-importer
+ */
+
+var gulp = require('gulp'),
+  merge = require('lodash.merge'),
+  utility = require('../utility');
+
+/**
+ * Function to get default options for an implementation of patternlab-import
+ * @returns {Object}  options  an object of default patternlab-import options
+ */
+var getDefaultOptions = function () {
```

Comment:
@scottnath I really think getDefaultOptions should be nested within gulpImportPatterns and the other globally scoped functions should be nested similarly. Just changing it to a variable assignment isn't taking it out of the global scope. In fact, the vars gulp, merge and utility are all in the global scope, but they are called from cache, and aren't intended to be mutable, so while some may argue whether it's a best practice to leave them there, there's really no possible of them causing harm. However, getDefaultOptions IS mutable in the global scope. That's why it shouldn't be there.